plugins/torrent/deluge_

```diff
@@ -0,0 +1,399 @@
+              ".info The number of connections used by Deluge Torrent")
+    elif mode == "bandwidth":
+        print("graph_title Bandwidth usage")
+        print("graph_order payloadDownloadRate overheadDownloadRate payloadUploadRate overheadUploadRate")
+        print("graph_args --base 1024 -r")
+        print("graph_vlabel bytes/s : down(-) and up(+)")
+        print("graph_scale yes")
+        print("graph_info This graph shows the bandwidth used by Deluge Torrent")
+        print("graph_category filetransfer")
+        print("graph_period second")
+
+        print("payloadDownloadRate.label payload")
+        print("payloadDownloadRate.draw AREA")
+        print("payloadDownloadRate.min 0")
+        print("payloadDownloadRate.graph no")
+        print("payloadDownloadRate.info Bandwidth used to download / upload torrents")
+
+        print("overheadDownloadRate.label overhead")
+        print("overheadDownloadRate.draw STACK")
+        print("overheadDownloadRate.min 0")
+        print("overheadDownloadRate.graph no")
+        print("overheadDownloadRate.info Bandwidth 'lost' due to overhead while downloading and uploading torrents")
+
+        print("payloadUploadRate.label payload")
+        print("payloadUploadRate.draw AREA")
+        print("payloadUploadRate.min 0")
+        print("payloadUploadRate.negative payloadDownloadRate")
+        print("payloadUploadRate.info Bandwidth used to upload torrents")
+
+        print("overheadUploadRate.label overhead")
+        print("overheadUploadRate.draw STACK")
+        print("overheadUploadRate.min 0")
+        print("overheadUploadRate.negative overheadDownloadRate")
+        print("overheadUploadRate.info Bandwidth 'lost' due to overhead while downloading and uploading torrents")
+    elif mode == "states":
+        print("graph_title Torrents states")
+
+        graph_order = " ".join(
+            [names_for_munin["state.{}".format(name)] for name in torrent_states])
+
+        print("graph_order " + graph_order)
+        print("graph_args --base 1000 -r --lower-limit 0")
+        print("graph_vlabel number of torrents")
+        print("graph_scale yes")
+        print("graph_info This graph shows the states of the torrents in Deluge Torrent")
+        print("graph_category filetransfer")
+        print("graph_period second")
+
+        for state_name in torrent_states:
+            print(names_for_munin["state." +
+                                  state_name] + ".label " + state_name)
+            print(names_for_munin["state." + state_name] + ".draw AREASTACK")
+            print(names_for_munin["state." + state_name] + ".type GAUGE")
+            print(names_for_munin["state." + state_name] + ".min 0")
+            print(names_for_munin["state." + state_name] +
+                  ".info Number of torrents in the '" + state_name + "' state")
+
+
+def fetch_info(mode):
+    if not successful_import:
+        print('Missing imports, cannot run !')
```

Comment:
You should write this to stderr - otherwise the munin server would try to interprete it.

Maybe import `print_function` from `__future__` and use `print(foo, file=sys.stderr)`?

Or `sys.stderr.write(foo + os.linesep)`?